RomanLoboda/HW6/task06_2.py

```diff
@@ -0,0 +1,14 @@
+input_login = input("Enter login : ")
```

Comment:
good job
look at this version of code
user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)